WordPressShared/WordPressShared/Core/Utility/NSString+Helpers.m

```diff
@@ -112,6 +112,14 @@ + (NSString *)stripShortcodesFromString:(NSString *)string
                                       withTemplate:@""];
 }
 
++ (NSString *)summaryFromContent:(NSString *)string
```

Comment:
@aerych, Doh! So it looks like there is one other spot this method is called:

https://github.com/wordpress-mobile/WordPress-iOS/pull/7572/files#diff-fab690f488774f7082db0c4344d1e60b

Thoughts?